spec/support/fixtures/cli_fixtures.cr

```diff
@@ -0,0 +1,20 @@
+module CLIFixtures
+  def expected_controller
+    <<-CONT
+class AnimalController < ApplicationController
+  def add
+    render("add.slang")
+  end
+
+  def list
+    render("list.slang")
+  end
+
+  def remove
+    render("remove.slang")
+  end
+end
+
+CONT
+  end
+end
```

Comment:
Crystal will create whitespace based on the closing `CONT`, so you can do:
```crystal
module CLIFixtures
  def expected_controller
    <<-CONT
    class AnimalController < ApplicationController
      def add
        render("add.slang")
      end
      def list
        render("list.slang")
      end
      def remove
        render("remove.slang")
      end
    end
    CONT
  end
end
```

Reply:
Yeah I was confused about this, fixing it

Reply:
The closing `CONT` should align with `class` and the last `end`